seleniumByTechnocredits/src/seleniumPractise/prachiJ/Assingment1/AlertDemo_Q1.java

```diff
@@ -0,0 +1,53 @@
+package seleniumPractise.prachiJ.Assingment1;
+
+import org.openqa.selenium.Alert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import util.Init;
+
+/*
+ * Alert Demo > Submit > Alert
+ * Validation of the values of Alert Demo with of the Alert.
+ */
+public class AlertDemo_Q1 {
+	public static void main(String[] args) throws InterruptedException {
+		WebDriver driver = Init.initChromeDriver();
+		String url = "http://automationbykrishna.com/#";
```

Comment:
As a optimized code,no need to take separate String as URL. You can directly use URL in driver.get()